src/actions/invitationActions.js

```diff
@@ -51,3 +51,46 @@ export function invitationsLoad() {
     });
   };
 }
+
+// invitationCreate
+// -----------------
+// Create a invitation
+// POST /invite/ to Passage.
+
+export function invitationCreate(invitation) {
+  return function(dispatch, getState) {
+    var token = getState().app.loggedInUser.auth.token;
+
+    var passage = new Passage({endpoint: window.config.passageEndpoint});
+
+    dispatch({type: types.INVITATION_CREATE});
+
+    return passage.createInvitation(token, invitation)
+    .then(result => {
+      dispatch({
+        type: types.INVITATION_CREATE_SUCCESS
+      });
+
+      dispatch(flashAdd({
+        message: 'Successfully invited ' + result.email,
+        class: 'success',
+        ttl: 3000
+      }));
+
+      dispatch(invitationsLoad());
+
+      return result;
+    })
+    .catch(error => {
+      console.error(error);
```

Comment:
is for debugging or you want to leave a message in the console?

Reply:
Yeah in this case I want to log the error, otherwise it gets swallowed completely. It's kind of the pattern i've been following so far.